cmd/init.go

```diff
@@ -130,7 +135,7 @@ dapr init --from-dir <path-to-directory>
 			if len(imageRegistryURI) != 0 {
 				warnForPrivateRegFeat()
 			}
-			err := standalone.Init(runtimeVersion, dashboardVersion, dockerNetwork, slimMode, imageRegistryURI, fromDir)
+			err := standalone.Init(runtimeVersion, dashboardVersion, dockerNetwork, slimMode, imageRegistryURI, fromDir, imageVariant)
```

Comment:
The parameter list keeps getting bigger, can we pass in a struct here?

Reply:
We have a `initInfo` struct in pkg/standalone (https://github.com/dapr/cli/blob/a9a03edf1c0f1a3d285d57df7b396b0d258e7190/pkg/standalone/standalone.go#L118), but it can't be used here due to `bundleDet` field. Adding another struct for passing parameters would be too similar to this